server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -239,7 +261,7 @@ void shouldRecomputeMailboxCounters(GuiceJamesServer server) throws MailboxExcep
 
         try {
             probe.appendMessage(BOB.asString(), inbox,
-                new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes(StandardCharsets.UTF_8)), new Date(), false, new Flags(Flags.Flag.SEEN));
+                new ByteArrayInputStream(MESSAGE_CONTENT_AS_BYTES), new Date(), false, FLAGS);
```

Comment:
Can we use `public ComposedMessageId appendMessage(String username, MailboxPath mailboxPath, MessageManager.AppendCommand appendCommand)` instead?